packages/style-value-parser/src/at-queries/__tests__/media-query-transform-test.js

```diff
@@ -182,6 +182,60 @@ describe('Media Query Transformer', () => {
     expect(JSON.stringify(result)).toBe(JSON.stringify(expectedStyles));
   });
 
+  test('basic usage: multiple heights', () => {
+    const originalStyles = {
+      foo: {
+        gridRow: {
+          default: '1 / 2',
+          '@media (max-height: 1200px)': '1 / 4',
+          '@media (max-height: 900px)': '1 / 3',
+          '@media (max-height: 600px)': '1 / -1',
+        },
+      },
```

Comment:
Missing example which are chains of `min-height` and also where the input has ranges already.